core/src/avm1/globals/system.rs

```diff
@@ -265,8 +265,8 @@ pub struct SystemProperties {
     /// If true then settings should be saved and read from the exact same domain of the player
     /// If false then they should be saved to the super domain
     pub exact_settings: bool,
-    /// If true then the system codepage should be used instead of unicode for text files
-    /// If false then unicode should be used
+    /// If true, the system codepage should be used for text files
+    /// If false, UTF-8 should be used for SWF version >= 6 and ISO Latin-1 for SWF version <= 5
```

Comment:
to be sure, the behavior on <=5 isn't the same as-if `useCodepage` was set to `true`?

Reply:
Yeah. I tested it on my macOS system and in a Windows VM with the codepage both set to German (Windows-1252) and Japanese (Shift-JIS).
When useCodepage has been set to true, the local codepage has always been used. But when I tried an SWFv5 file, it has always been encoded as Windows-1252 (to be exact, a custom but similar encoding on macOS), even when the codepage was Japanese.

In this image you can see the same code loading some texts in different encodings on a Windows system with a Japanese codepage. The only difference is
SWFv10 / SWFv10 with useCodepage true / SWFv5
<img width="1023" alt="Bildschirmfoto 2024-06-19 um 01 52 02" src="https://github.com/ruffle-rs/ruffle/assets/33464213/dfb4a49f-0453-441f-a1f0-e1ac7f55b76a">